lib/paper_trail.rb

```diff
@@ -101,7 +101,8 @@ def self.serializer
   end
 
   def self.active_record_protected_attributes?
-    @active_record_protected_attributes ||= ::ActiveRecord::VERSION::MAJOR < 4 || !!defined?(ProtectedAttributes)
+    @active_record_protected_attributes ||= ::ActiveRecord::VERSION::MAJOR < 4 ||
+      !defined?(ProtectedAttributes).nil?
```

Comment:
I actually think `!!defined?(ProtectedAttributes)` is clearer than the double negative of `!defined?(ProtectedAttributes).nil?`